app/assets/javascripts/spotlight/block_mixins/titleable.js

```diff
@@ -1,12 +1,12 @@
-(function ($){
-  SirTrevor.BlockMixins.Titleable = {
-    mixinName: "Titleable",
-
-    initializeTitleable: function() {
-      this.$inner.append("<div class='st-title'>" + this.title() + "</div>");
-    },
-  }
-  
-  SirTrevor.Block.prototype.availableMixins.push("titleable");
-  SirTrevor.Block.prototype.titleable = true;
-})(jQuery);
+// (function ($){
+//   SirTrevor.BlockMixins.Titleable = {
```

Comment:
Part of the WIP. Need to discuss with @ggeisler about how we want to restyle the controls in light of upstream changes.